liferay-echo/liferay-portal
# LPS-187659 The AI

Aaaaand now it works!

This first four commits have been sent to frontend infra repo and will be reviewed there: https://github.com/liferay-frontend/liferay-portal/pull/3452

Apart from that, this PR has just some final touches to make everything work as expected:

![Peek 2023-06-15 13-23](https://github.com/liferay-echo/liferay-portal/assets/800645/ac740696-7f1c-4b99-98d5-68249ac69dfb)

![Peek 2023-06-15 13-21](https://github.com/liferay-echo/liferay-portal/assets/800645/4d8b07bc-2974-4966-a473-701b9f1c5229)

In order to test this:
- Enable feature flag LPS-179483
- You should see the "AI Creator" button in web content creation, but it will show just a popover.
- Go to `Instance Settings > AI Creator` and set an API key (we have some inside our team vault).
- Go back to web content creation and it should work fine.
